api/src/services/telemetry-services/telemetry-vendor-service.ts

```diff
@@ -15,6 +15,8 @@ import { TelemetryDeploymentService } from './telemetry-deployment-service';
 /**
  * A service class for working with telemetry vendor data.
  *
+ * @see https://api.vectronic-wildlife.com/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config#//getPositions_2
```

Comment:
Is there a better place for this link? Seems slightly random. Somewhere in one of the cronjob files would be better I think.

Reply:
Looks like this link is already in cronjob.ts

Reply:
Dropped